tests/foreman/ui/test_host.py

```diff
@@ -513,6 +519,128 @@ def test_positive_delete(self):
             self.hosts.delete(
                 u'{0}.{1}'.format(host.name, host.domain.name))
 
+    @run_only_on('sat')
+    @tier2
+    def test_positive_search_by_parameter(self):
+        """Search for the host by global parameter assigned to it
+
+        @id: 8e61127c-d0a0-4a46-a3c6-22d3b2c5457c
+
+        @expectedresults: Only one specific host is returned by search
+
+        @CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        param_name = gen_string('alpha')
+        param_value = gen_string('alpha')
+        parameters = [{'name': param_name, 'value': param_value}]
+        param_host = entities.Host(
+            organization=org,
+            host_parameters_attributes=parameters,
+        ).create()
+        additional_host = entities.Host(organization=org).create()
```

Comment:
It is a little bit redundant, but if you think that is necessary - I can do that